proto/session_server.proto

```diff
@@ -51,4 +68,5 @@ message SessionResult {
  */
 service SessionServer {
   rpc StartSession(SessionParams) returns (stream SessionProgress) {}
+  rpc ConfigureEndpoint(TunnelEndpoint) returns (SessionResult) {}
```

Comment:
Is this extra? it's also not defined anywhere.

Reply:
removed